wasm-build/wordpress-data/prepare-wordpress.sh

```diff
@@ -114,6 +61,11 @@ done
 
 # Let the WordPress installer do its magic
 cp wp-config-sample.php wp-config.php # Required by the drop-in SQLite integration plugin
+
+# Disable load-scripts.php
+sed "s/<?php/<?php define( 'CONCATENATE_SCRIPTS', false );/" wp-config.php > wp-config.php.new
+mv wp-config.php.new wp-config.php
```

Comment:
I'd move this to wordpress.mjs just because all the other WordPress customizations live there – I'd like to avoid the fragmentation.

Reply:
Hm, on the other hand this is a build feature. You can't remove assets without setting `CONCATENATE_SCRIPTS` to false. Alright – good call putting it here!